main.rb

```diff
@@ -0,0 +1,52 @@
+require_relative './app'
+
+@app = App.new
+
+def list_options
+  puts '
+  Please choose one of the options: (1 - 7)
+    1. - List all books
+    2. - List all people
+    3. - Create a person
+    4. - Create a book
+    5. - Create a rental
+    6. - List all rentals for a given person id
+    7. Exit
+  '
+end
+
+def start_program
+  loop do
+    list_options
+    choice = gets.chomp.to_i
+    if choice.between?(1, 7)
+      if choice == 7
+        puts 'Welldone for using this app'
+        exit
+      else
+        choose_operation(choice)
+      end
+    end
+  end
+end
+
+def choose_operation(opt)
+  case opt
+  when 1
+    @app.list_books
+  when 2
+    @app.list_peoples
+  when 3
+    @app.create_person
```

Comment:
- [ ] Creating a person gives an error kindly consider fixing this so your app is functional

![Screenshot_20221104_181533](https://user-images.githubusercontent.com/25227022/200036711-aef5b870-2e42-41bf-a93a-b288fcc79455.png)

Reply:
@Ismailco Thank you for this review